decidim-core/app/helpers/decidim/layout_helper.rb

```diff
@@ -129,28 +129,6 @@ def extended_navigation_bar(items, max_items: 5)
       }
     end
 
-    # Renders a view with the customizable CSS variables in two flavours:
-    # 1. as a hexadecimal valid CSS color (ie: #ff0000)
-    # 2. as a disassembled RGB components (ie: 255 0 0)
-    #
-    # Example:
-    #
-    # --primary: #ff0000;
-    # --primary-rgb: 255 0 0
-    #
-    # Hexadecimal variables can be used as a normal CSS color:
-    #
-    # color: var(--primary)
-    #
-    # While the disassembled variant can be used where you need to manipulate
-    # the color somehow (ie: adding a background transparency):
-    #
-    # background-color: rgba(var(--primary-rgb), 0.5)
-    def organization_colors
```

Comment:
@andreslucena , i believe removing this method is outside the scope of this backport. Could you revert?